data/movements/movements.xml

```diff
@@ -35,95 +35,185 @@
 
 	<!-- (Level & quest) doors -->
 	<movevent event="StepOut" itemid="1224" script="closingdoor.lua" />
+	<movevent event="StepIn" itemid="1224" script="opendoor.lua" />
```

Comment:
my fault, it was named opendoor, but then i changed to enterdoor.

Ill fix it tomorrow. Anything else you want me to change?